openghg/store/_obssurface.py

```diff
@@ -363,6 +364,20 @@ def read_file(
                 for key, value in data.items():
                     data[key]["data"] = value["data"].chunk(chunks)
 
+            for _, gas_data in data.items():
```

Comment:
I think it would be worth making this added section a method on the `ObsSurface` class e.g. `align_metadata_attributes(...)`

In future would be good to create a more generic function for this functionality on the `BaseStore` class which would then be applied to all of the data type classes. Splitting this out now should make this easier to use as a blueprint when we're able to do this.